astroid/bases.py

```diff
@@ -620,16 +620,16 @@ def callable(self):
     def pytype(self) -> Literal["builtins.generator"]:
         return "builtins.generator"
 
-    def display_type(self):
+    def display_type(self) -> str:
```

Comment:
We could have used `Literals` here. Not sure if it makes sense to change them now though...

Reply:
Yeah, we have a lot to do before starting to polish this to use litteral imo :D

Reply:
For future PRs I do think we can start using them, if they make sense. Sometimes it's useful to know the specific string a function returns 😄

Reply:
> For future PRs I do think we can start using them, if they make sense. Sometimes it's useful to know the specific string a function returns 😄

I would disagree. We should only use `Literal` if there is a clear value to it. In most cases however it just adds unnecessary complexity. Example, what if the return string changes? You'll also have to update the `Literal` typing as well.

`Literal` is most useful for `@overloads` IMO.

Reply:
I think for many of the methods that were being typed here the method could have also been a attribute, in which case I think it makes sense to do so as you would get similar information about the return value if it was an attribute.

But you are right, they shouldn't be used everywhere.